gateway/api_definition.go

```diff
@@ -558,7 +558,16 @@ func (a APIDefinitionLoader) processRPCDefinitions(apiCollection string, gw *Gat
 			def.Proxy.ListenPath = newListenPath
 		}
 
-		spec := a.MakeSpec(def, nil)
+		spec := a.MakeSpec(def.APIDefinition, nil)
+		if spec.IsOAS {
+			loader := openapi3.NewLoader()
+			if err := loader.ResolveRefsIn(&def.OAS.T, nil); err != nil {
+				log.WithError(err).Errorf("RPC loaded API's OAS reference resolve failed: %s", def.APIID)
```

Comment:
Should we return an error here?

Reply:
no, the other ones should continue to be loaded

Reply:
ok, makes sense